### DISCUSSION TITLE
Not able to open PP-ChatOCRv2 link https://aistudio.baidu.com/application/detail/10368

### DISCUSSION THREAD

**Discussion by suresh-saini** [PaddlePaddle/PaddleOCR#12293] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：NA
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：NA
- 完整报错/Complete Error Message：Link given on github is not working

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
I am trying to open PP-ChatOCRv2 link  https://aistudio.baidu.com/application/detail/10368 given in the repository document but i am getting blank screen only. can some one help me how can I explore the PP-ChatOCRv2?

![Screenshot from 2024-04-04 15-15-38](https://github.com/PaddlePaddle/PaddleOCR/assets/72432228/755a8dc1-7afc-4eab-9764-50fc9551a1f9)

**Comment by cuicheng01** [comment]
Welcome to experience the new version of PP-ChatOCRv2, you can browse this [link](https://aistudio.baidu.com/pipeline/mine) , and then create a development pipeline for PP-ChatOCRv2.

**Comment by suresh-saini** [comment]
Thanks @cuicheng01. but even the link you shared is not working for me 
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/72432228/7ced4388-869b-483c-a10d-03a5d647266d)

**Comment by cuicheng01** [comment]
hi，can you register an aistudio account?
